plugins/@grouparoo/demo/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "author": "Grouparoo Inc <hello@grouparoo.com>",
+  "name": "@grouparoo/demo",
+  "description": "The Grouparoo Demo Setup Plugin",
+  "version": "0.1.12-alpha.1",
+  "license": "MPL-2.0",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
Maybe we don't actually want to publish this one?  
Lerna knows not to publish things that are `private` (defined here)

Reply:
maybe. I didn't do it on purpose per se, but I can imagine a world where someone wants to load it up with data. doesn't seem to hurt.

Reply:
whichever you think.

Reply:
I'm indifferent at the moment - just calling out the options we have.   It'll get published tonight!